### PR TITLE
Add test for Migration Helper (#518)

### PR DESCRIPTION
**Pull request by mbelsky**
Hey,
There are a few tests for Migration Helper.

### CODE REVIEW

File: __tests__/tests/utilities/migration-helper.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import {getRepoIdFromUrl, toOldIssueFormat, toOldUserFormat} from 'utils'
```
Comment (lex111) [inline __tests__/tests/utilities/migration-helper.js]: Can you add spaces between curly braces?

```js
import { getRepoIdFromUrl, toOldIssueFormat, toOldUserFormat } from 'utils'
```

File: __tests__/tests/utilities/migration-helper.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+describe('Migration Helper', () => {
+  it('should return a repo id in {username}/{reponame} format', () => {
+    const result = getRepoIdFromUrl('https://api.github.com/repos/octocat/Hello-World');
+    expect(result).toBe('octocat/Hello-World');
```
Comment (lex111) [inline __tests__/tests/utilities/migration-helper.js]: Here and further, please add a new line before `expect`.

File: __tests__/tests/utilities/migration-helper.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      comments: {
+        totalCount: 7,
+      },
+      closedAt: '',
+      createdAt: '',
+      number: 42,
+      state: 'StAte',
+    };
+    const repoId = 'git-point';
+
```
Comment (lex111) [inline __tests__/tests/utilities/migration-helper.js]: Why is there a space? Let's add one a space between the variable declarations and the `expect` calls, and stick with it in all `it` blocks, what do you think?

```js
const a = 'something';
const b = 'something else';

expect(...);
expect(...);
```

**Comment by mbelsky** [comment]
@lex111 Thanks for review, I've fixed that.

**Comment by lex111** [comment]
@mbelsky it's awesome, thanks! Happy Hacktoberfest! :jack_o_lantern:

**Comment by housseindjirdeh** [comment]
🎃

**Pull request merged**